installer/uninstaller.sh

```diff
@@ -38,30 +38,60 @@ CHARTS_LIST="$RELEASE_PREFIX-mgt-api $RELEASE_PREFIX-crd $RELEASE_PREFIX-dex $RE
 HELM_LS_OUTPUT=`helm ls --output json`
 HELM_LIST=`jq --arg namearg "Releases" '.[$namearg]' <<< $HELM_LS_OUTPUT`
 K8S_NS_OUTPUT=`kubectl get ns --output=json`
-K8S_NS_LIST=`jq --arg namearg "items" '.[$namearg]' <<< $K8S_NS_OUTPUT`
+K8S_NAMESPACES=`echo $K8S_NS_OUTPUT | jq '.items[].metadata.name'`
 helm_arr=()
 K8S_NS_ARR=()
 
+[[ $K8S_NAMESPACES =~ "default-tenant" ]] && SINGLE_TENANT=1 || SINGLE_TENANT=0
```

Comment:
so if there is only one signle tenant called default-tenant we say that the platform is in SINGLE_TENANT mode. A little assumption. @joannanosek recently in #56 add special variable to determine if we use Single Tenant Mode or not - DEFAULT_TENANT_NAME. We should be consistent weather we are making assumption or not.

Reply:
I'm not even sure if we must necessarily have all of these checks. As I suggested below, just use `./imm ls t` to distinguish tenants.